tests/Oryx.Integration.Tests/LocalDockerTests/NodeEndToEndTests.cs

```diff
@@ -179,10 +179,8 @@ public async Task CanBuildAndRunNodeApp_OnSecondBuild_AfterZippingNodeModules_In
 
             var buildCommand = $"oryx build {appDir} -i /tmp/int -o /tmp/out -l nodejs --language-version {nodeVersion}";
             var buildScript = new ShellScriptBuilder()
-                .AddCommand("export ORYX_ZIP_NODE_MODULES=true")
-                .AddCommand(buildCommand)
-                .AddCommand("export ORYX_ZIP_NODE_MODULES=false")
-                .AddCommand(buildCommand)
+                .AddCommand($"oryx build {appDir} -i /tmp/int -o /tmp/out -l nodejs --language-version {nodeVersion} -p compress_node_modules=tar-gz")
+                .AddCommand($"oryx build {appDir} -i /tmp/int -o /tmp/out -l nodejs --language-version {nodeVersion}")
```

Comment:
What would be the state of toml file after the second build?

Reply:
it will be deleted

Reply:
and not generated again since there isn't any property to be stored